src/app/account/repo/checks/[id]/form.js

```diff
@@ -3,11 +3,14 @@
 import {
   DocumentDuplicateIcon,
   CheckBadgeIcon,
```

Comment:
I think `CheckBadgeIcon` is no longer used.

Don't worry about fixing it, I will merge to a temporary branch and fix now, so we can get this feature merged